app/webpacker/styles/_custom.scss

```diff
@@ -5,3 +5,16 @@
 .govuk-green {
   color: govuk-colour("green");
 }
+
+.app-contents {
+  list-style-type: none;
+  padding: 0;
+  margin: 0 0 0 2.2em;
+  margin-bottom: 0;
+  text-indent: -2.2em;
+}
+
+.app-contents li::before {
+  content: "\2014";
```

Comment:
Where is exactly needed?

Reply:
I actually don't know. It came with Matt's heroku review prototype